site/_docs/reference.md

```diff
@@ -3542,6 +3544,34 @@ dropFunctionStatement:
 truncateTableStatement:
       TRUNCATE TABLE name
       [ CONTINUE IDENTITY | RESTART IDENTITY ]
+
+grantStatement:
+      GRANT
+      {
+          privilege [, privilege ]*
+      |   ALL [ PRIVILEGES ]
+      }
+      ON
+      {
+          [ TABLE ] table [, table ]*
+      |   ALL TABLES IN SCHEMA schema [, schema ]*
+      |   ALL TABLES IN ROOT SCHEMA
+      }
+      TO user [, user ]*
+
+revokeStatement:
+      REVOKE
+      {
+          privilege [, privilege ]*
+      |   ALL [ PRIVILEGES ]
+      }
+      ON
+      {
+          [ TABLE ] table [, table ]*
```

Comment:
1. Are only the `TABLE` type currently considered for resource types? Is this syntax more generic, such as using `resource_type`, which lists supported types: `TABLE`
2. Can `ALL TABLES IN SCHEMA` be equivalently represented as the  `SCHEMA` resource type?